chart/environment-values/values.base.insecure.yaml

```diff
@@ -106,4 +92,4 @@ kafka:
   broker:
     persistence:
       size: 20Gi
-    replicaCount: 2
+    replicaCount: 0
```

Comment:
Should we just remove `broker` section?

Reply:
no, let's leave it, It should be an option in HA setups.

Reply:
Need to put a comment somewhere about it.